index.js

```diff
@@ -1 +1,37 @@
 // implement your API here
+const express = require("express");
+const db = require("./data/db.js");
+
+const server = express();
+
+server.use(express.json());
+
+server.post("/api/users", (req, res) => {
+  const newUser = req.body;
+  console.log("newUser", newUser);
+  db.user
+
+    .add(user)
+    .then(user => {
+      res.status(201).json({ success: true, user });
+    })
+
+    .catch(({ code, message }) => {
+      res.status(code).json({ success: false, message });
+    });
+});
+
+server.get("/api/users", (req, res) => {
+  db.users
```

Comment:
Same issue as above, there is no `users` object in the db exports that contains a function.